### DISCUSSION TITLE
Should we create a testing tools package to easily create tests?

### DISCUSSION THREAD

**Discussion by LaiaPerez88** [boostercloud/booster#649] [Meta]
Create a testing tools package to let developers easily create tests. Let's brainstorm!

**Comment by davidverdu** [comment]
Can we have more details for this issue implementation @boostercloud/booster-core ? Maybe creating an external repository with these testing functions? these functions should help programmer testing Booster abstractions?

**Comment by NickSeagull** [comment]
I think this fits more a discussion than an issue, as its more for brainstorming and coming up with ideas. Converting into a discussion

**Comment by NickSeagull** [comment]
Theres booster-testing-utils somewhere, closing
